kn/leden/mongo.py

```diff
@@ -33,9 +38,20 @@ def delete(self):
     def save(self, update_fields=NotImplemented):
         if self._parent is None:
             if '_id' in self._data:
-                self._collection.update({
-                    '_id': self._data['_id']}, self._data)
+                if self._versioned:
+                    self._data['_version'] += 1
+                    result = self._collection.update({
+                        '_id': self._data['_id'],
+                        '_version': self._data['_version']-1}, self._data, w=1)
+                    if result['n'] < 1:
+                        raise MongoWrapperException(
+                            'Could not save - version mismatch?')
```

Comment:
Kan. De naam van de exception verander ik wel.
In het geval van de events lijkt me dat wel overdreven: het is meer een extra zekerheid voor het geval dat.